pkg/utils/ocmWrapper.go

```diff
@@ -100,6 +101,26 @@ func (*DefaultOCMInterfaceImpl) GetOCMAccessToken() (*string, error) {
 	return &accessToken, nil
 }
 
+func (*DefaultOCMInterfaceImpl) GetPullSecret() (string, error) {
+
+// Get ocm access token
+	logger.Debugln("Finding ocm token")
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		return "", fmt.Errorf("failed to create OCM connection: %v", err)
+	}
+	defer connection.Close()
+	response, err := connection.Post().Path("/api/accounts_mgmt/v1/access_token").Send()
```

Comment:
You can use SDK to get the access token `connection.AccountsMgmt().V1().AccessToken().Post().Send()`

Reply:
I tried to change this to below but got this error and was not able fix, so keeping it as is for now. 
```
cannot use response.Body() (value of type *"github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1".AccessToken) as io.Reader value in argument to ioutil.ReadAll: *"github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1".AccessToken does not implement io.Reader (missing method Read)compiler
in bodyBytes, err := ioutil.ReadAll(response.Body())
```
```
func (*DefaultOCMInterfaceImpl) GetPullSecret() (*v1.AccessTokenPostResponse, error) {
    // Get ocm access token
    logger.Debugln("Finding ocm token")
    connection, err := ocm.NewConnection().Build()
    if err != nil {
        return nil, fmt.Errorf("failed to create OCM connection: %v", err)
    }
    defer connection.Close()

    response, err := connection.AccountsMgmt().V1().AccessToken().Post().Send()
    if err != nil {
        return nil, fmt.Errorf("failed to get pull secret from ocm: %v", err)
    }
    defer response.Body().Close()

    bodyBytes, err := ioutil.ReadAll(response.Body())
    if err != nil {
        return nil, fmt.Errorf("failed to read response body: %v", err)
    }

    var pullSecret v1.AccessTokenPostResponse
    err = json.Unmarshal(bodyBytes, &pullSecret)
    if err != nil {
        return nil, fmt.Errorf("failed to decode response body: %v", err)
    }

    logger.Debugln("Found pull secret from ocm")
    return &pullSecret, nil
}

```